lib/vehicle_factory.rb

```diff
@@ -0,0 +1,37 @@
+require './lib/vehicle'
+
+class VehicleFactory
```

Comment:
Yes, nice work here with this Vehicle Factory. This is how the Facility factory should be working as well, just taking in one giant set of data and then filtering through it.